src/components/Post/Comment/CommentPanel.tsx

```diff
@@ -71,24 +72,32 @@ const CommentPanel = ({ post }: { post: Post | undefined }) => {
   }
 
   return (
-    <Box flexDir={'column'} margin={2} position='fixed' width={'22%'}>
+    <Box flexDir={'column'} top='100px' position='fixed' width={'25%'} height='100%'>
       {post !== undefined ?
-        <Box>
-          <Text alignSelf={'center'}>Comments</Text>
-          {comments === undefined || comments!.length === 0 ?
+        <>
+          <Box height='25px'>
+            <Text alignSelf={'center'} bold>Comments</Text>
+          </Box>
+          {comments === undefined || comments.length === 0 ?
             <Box marginTop={2}><Text alignSelf={'center'}>No comments just yet.</Text></Box>
             :
-            comments?.map((value, index) => {
-              return (
-                <Box key={index} margin={2}>
-                  <CommentDisplay comment={value} />
-                </Box>
-              );
-            })
-          }
-          {hasMoreComments && 
-                        <Button onPress={fetchMoreComments}><Text variant='button'>Load More</Text></Button>}
-        </Box> :
+            <div className='comment-panel-container'>
+              <div className='comment-panel'>
+                {comments?.map((value, index) => {
+                  return (
+                    <Box key={index} margin={1}>
```

Comment:
Wait aren't we supposed to use the docrefID for keys in maps?

Reply:
You're 100% right, thanks for catching that. I took it out for a moment because when no post was selected it would try to access the docRef id and crash, but that's been fixed.